cmd/porter/bundle.go

```diff
@@ -168,23 +104,14 @@ For instance, the 'debug' driver may be specified, which simply logs the info gi
 	return cmd
 }
 
-func buildInstallCommand(p *porter.Porter) *cobra.Command {
-	cmd := buildBundleInstallCommand(p)
-	cmd.Example = strings.Replace(cmd.Example, "porter bundle install", "porter install", -1)
-	cmd.Annotations = map[string]string{
-		"group": "alias",
-	}
-	return cmd
-}
-
 func buildBundleUpgradeCommand(p *porter.Porter) *cobra.Command {
 	opts := porter.UpgradeOptions{}
 	cmd := &cobra.Command{
-		Use:   "upgrade [CLAIM]",
+		Use:   "upgrade [INSTANCE]",
```

Comment:
line 111 below: perhaps update to `Upgrade a bundle instance`?  (Would apply similarly across this file)